src/main/java/aztech/modern_industrialization/machines/blockentities/AbstractStorageMachineBlockEntity.java

```diff
@@ -91,10 +89,6 @@ public MIInventory getInventory() {
         return MIInventory.EMPTY;
     }
 
-    public static MachineCasing getCasingFromTier(CableTier from, CableTier to) {
```

Comment:
I think the MachineCasing should be a property of the CableTier. MachineCasings are already easy to register in the KubeJS API so this won't be a problem.